app/ui/src/main/res/xml/account_settings.xml

```diff
@@ -46,6 +46,11 @@
             android:summary="@string/account_settings_mark_message_as_read_on_view_summary"
             android:title="@string/account_settings_mark_message_as_read_on_view_label" />
 
+        <CheckBoxPreference
```

Comment:
The other settings related to deletion are under 'Fetching mail'. That's not great either, but I suggest we move it there for now.